pages/common/ssh-copy-id.md

```diff
@@ -2,6 +2,14 @@
 
 > Install your public key in a remote machine's authorized_keys.
 
+- Copy your keys to the remote machine:
+
+`ssh-copy-id {{username@remote_host}}`
+
+- Copy your default key to the remote machine:
```

Comment:
This is not useful IMO. Copying your default key with ssh would require access to the machine in the first place. Which means you have to give password auth or some other key.